set-transaction.md

```diff
@@ -78,7 +78,8 @@ You can also set the default isolation level for all transactions in the client'
 
 ## See Also
 
-- [Transactions](transactions.html)
```

Comment:
This document still needs a link to `transactions.html`; I'd recommend putting it up in the first paragraph. Up there you could also introduce the notion of "transaction parameters."